graphene_sqlalchemy/filters.py

```diff
@@ -158,7 +187,9 @@ def execute_filters(
                 )
                 clauses.extend(_clauses)
             else:
-                model_field = getattr(model, field)
+                model_field = getattr(
+                    model, input_field.model_attr
+                )  # getattr(model, field)
```

Comment:
Is it possible for `model_field` to be `None` here? Looking at [L#486 of types.py](https://github.com/graphql-python/graphene-sqlalchemy/blob/322f12a1523b4866bbc01bc21724281c8707d055/graphene_sqlalchemy/types.py#L486), couldn't a `graphene.InputField` be passed in to a `BaseTypeFilter`? Not sure if I understand the scope of this change though.